lib/web_ui/dev/test_runner.dart

```diff
@@ -60,6 +60,10 @@ class TestCommand extends Command<bool> {
 
     _copyTestFontsIntoWebUi();
     await _buildHostPage();
+    if(io.Platform.isWindows) {
+      // Error on windows if not run.
```

Comment:
Might be worth leaving links to the Dart SDK issues you mentioned in the PR description

Reply:
done.